cmd/rollkit/docs/rollkit_start.md

```diff
@@ -33,7 +33,7 @@ rollkit start [flags]
       --rollkit.da_address string                       DA address (host:port) (default "http://localhost:26658")
       --rollkit.da_auth_token string                    DA auth token
       --rollkit.da_block_time duration                  DA chain block time (for syncing) (default 15s)
-      --rollkit.da_gas_multiplier float                 DA gas price multiplier for retrying blob transactions (default -1)
+      --rollkit.da_gas_multiplier float                 DA gas price multiplier for retrying blob transactions
```

Comment:
do you still want to say `default 0`?

Reply:
It seems to be autogenerated. I think it's fine.